nautobot/docs/configuration/required-settings.md

```diff
@@ -285,56 +287,6 @@ The following environment variables may also be set for some of the above values
 
 For more details on configuring RQ, please see the documentation for [Django RQ installation](https://github.com/rq/django-rq#installation).
 
-#### Using Redis Sentinel
-
-If you are using [Redis Sentinel](https://redis.io/topics/sentinel) for high-availability purposes, you must be using dictionary-style settings, and modify the connection settings. This requires the removal of the `HOST`, `PORT`, and `DEFAULT_TIMEOUT` keys from the example above and the addition of three new keys.
```

Comment:
The information in this paragraph (specifically on keys that should NOT be part of the `RQ_QUEUES` dictionary) seems to no longer be present. Is it still relevant? Or is the idea with the new docs that you do *not* need to use "dictionary-style" `RQ_QUEUES` settings but instead should now use the basic `USE_REDIS_CACHE` pattern?

Reply:
since `RQ_QUEUES` uses `"USE_REDIS_CACHE": "default"` now I didn't want to get in to configuring `RQ_QUEUES` separately from `CACHES`.  I also didn't really want to explain what keys were added/removed in detail other than the config for sentinel is different here's an example with some explanation of the interesting keys.